agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -2321,9 +2321,6 @@ bool backhaul_manager::handle_1905_1_message(ieee1905_1::CmduMessageRx &cmdu_rx,
     case ieee1905_1::eMessageType::AP_CAPABILITY_QUERY_MESSAGE: {
         return handle_ap_capability_query(cmdu_rx, src_mac);
     }
```

Comment:
the commit message should follow the `contribution.md` rules.

Please use this PR to ramp-up on them.
(just from a quick review: line-length limits of title and body are not met)

Reply:
fixed